README.md

```diff
@@ -1,47 +1,97 @@
+* Make sure you have an X11 server running.
+    * On Ubuntu Linux, it should be running by default.
+    * On macOS, Install and launch [XQuartz](https://www.xquartz.org/) (make sure to go to XQuartz > Preferences > Security and check "Allow client connections").
+* Install [VScode](https://code.visualstudio.com/download)
+* Open a terminal and clone `app-boilerplate` with `git clone git@github.com:LedgerHQ/app-boilerplate.git`.
+* Open the `app-boilerplate` folder with VSCode. VSCode should prompt you to install the "remote development" extension. Otherwise, install the extension manually.
+* Choose the [devcontainer](https://code.visualstudio.com/docs/devcontainers/containers) for your platform with `ctrl + shift + p` (`command + shift + p` on a Mac) and typing "Rebuild and Reopen in Container".
+* Wait for the [ledger-app-dev-tools]() docker image to be pulled...
+* Select the device to build for with `ctrl + shift + b` (`command + shift + b` on a Mac) and select `Run [debug] make`.
+* Test your binary on [Speculos](https://github.com/LedgerHQ/speculos) with `ctrl + shift + b` (`command + shift + b` on a Mac) and select task `Run Speculos`.
+
+**Notes**
+
+1. You can find the tasks definitions in `.vscode/tasks.json`
+2. You can find the devcontainers configuration files in the `.devcontainer` directory.
+
+**Windows users**
+
+### TODO WRITE THIS SECTION OR INTEGRATE IT IN THE PREVIOUS SECTION
+
+### With a shell and the docker builder image
 
 The app-builder docker image [from this repository](https://github.com/LedgerHQ/ledger-app-builder) contains all needed tools and library to build and load an application.
 You can download it from the ghcr.io docker repository:
 
 ```shell
-sudo docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+sudo docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
 ```
 
 You can then enter this development environment by executing the following command from the directory of the application `git` repository:
 
 ```shell
-sudo docker run --rm -ti --user "$(id -u)":"$(id -g)" -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+sudo docker run --rm -ti --user "$(id -u)":"$(id -g)" -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder
 ```
 
 The application's code will be available from inside the docker container, you can proceed to the following compilation steps to build your app.
 
-### Without the docker image builder
+## Compilation and load
 
-Be sure to have your environment correctly set up (see [Getting Started](https://developers.ledger.com/docs/nano-app/introduction/)) and [ledgerblue](https://pypi.org/project/ledgerblue/) and installed.
+```shell
+make DEBUG=1  # compile optionally with PRINTF
+make load     # load the app on the Nano using ledgerblue
+```
+
+You can choose which device to compile and load for by setting the `BOLOS_SDK` and `TARGET` environment variables to the following values :
```

Comment:
TARGET should not be needed when SDK is explicit.

Reply:
Indeed. Will update it.

Reply:
Indeed. Will update it.